sidebars.js

```diff
@@ -26,6 +26,7 @@ module.exports = {
         "features/workspaces",
         "features/repo-insights",
         "features/contributor-insights",
+        "features/connections",
```

Comment:
I intentionally removed this from the sidebar.

Reply:
I'll change it back 👍